### PR TITLE
Include TRNG 4.22 updates

### LINKED ISSUE #6: Upgrade to TRNG 4.22
An updated version of [TRNG](https://www.numbercrunch.de/trng/) has been released:

- **Version 4.21, May 11, 2019**:  Bug-fix and service release. Fixes numerical convergence problems in the inverse of the incomplete Beta function.
- **Version 4.22, August 28, 2019**: This maintenance release removes old code for supporting C++ language standards older than C++11. Many minor code enhancements and bug fixes have been applied. The autotools-based build system has been replaced by CMake to modernize the build process and enhance portability, see installation instructions. Many minor code enhancements and bug fixes. The negative binomial distribution has been generalized to real-valued parameters

rTRNG should be upgraded (using [inst/tools/upgradeTRNG.R](../blob/74247fae329efbae760e954fb2cd730b7f2c1329/inst/tools/upgradeTRNG.R), which may require an update).

### PR DESCRIPTION
**Pull request by riccardoporreca**
Closes #6 (see for details)

### CODE REVIEW

File: src/Makevars [hunk 1]
```diff
@@ -3,6 +3,8 @@ PKG_LIBS += $(shell ${R_HOME}/bin/Rscript \
               -e "RcppParallel::RcppParallelLibs()")
 
 PKG_CPPFLAGS += -I../inst/include
+# path for the direct includes in TRNG sources from version 4.22
```
Comment (RolandASc) [inline src/Makevars]:
```suggestion
# path for the direct includes in TRNG sources from version 4.22 onwards
```

File: src/Makevars.win [hunk 2]
```diff
@@ -5,6 +5,8 @@ PKG_LIBS += $(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" \
               -e "RcppParallel::RcppParallelLibs()")
 
 PKG_CPPFLAGS += -I../inst/include
+# path for the direct includes in TRNG sources from version 4.22
```
Comment (RolandASc) [inline src/Makevars.win]:
```suggestion
# path for the direct includes in TRNG sources from version 4.22 onwards
```

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # rTRNG 4.20-1.9000
```
[the remark this answers is not in the document]
Comment (riccardoporreca) [reply NEWS.md]: Absolutely:
```suggestion
# rTRNG 4.22-0.9000
```

**Comment by riccardoporreca** [comment]
> * I would maybe consider using our fork of r-appveyor, but I understand that wouldn't be ideal either

We can reconsider this after merging

**Pull request merged**